Optional CreateDatabase upgrade

luisfurnas opened the pull request:
Add createDatabase boolean param to init for optional CreateDatabase upgrade

darkfrog26 commented:
I like the change, but I don't care for the ScalaFMT changes. If you are willing to get rid of that or change the file to match my formatting, I'd be happy to approve this.

luisfurnas commented:
Removed the ScalaFMT file, and reverted all formatting changes to be exactly how it was.
Changed my IDE to apply IntelliJ formatting, so the formatting was reverted to how it was; however the comments formatting was slightly changed. Is it okay, or should I also revert the comments formatting?
